scripts/utils/verify_scripts.js

```diff
@@ -1,8 +1,8 @@
 module.exports = function (web3 = web3, artifacts = artifacts) {
   const assert = require("assert")
   const Contract = require("@truffle/contract")
-  const { getOrdersPaginated } = require("@gnosis.pm/dex-contracts/src/onchain_reading")
-  const { Fraction } = require("@gnosis.pm/dex-contracts/src")
+  const { getOrdersPaginated } = require("@gnosis.pm/dex-contracts/onchain_reading")
+  const { Fraction } = require("@gnosis.pm/dex-contracts")
```

Comment:
```suggestion
  const { Fraction, getOrdersPaginated } = require("@gnosis.pm/dex-contracts")
```

Reply:
Thanks! So I take it everything it now imported this way.

Reply:
All the re-exported methods all. Things like the migration scripts which aren't re-exported still need to be imported with the full path.